ChangeLog.d/fix-psa-cmac.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix auto-enabling of MBEDTLS_CIPHER_C when builtin CMAC is enabled in PSA
```

Comment:
“Fix auto-enabling” makes me think that we used to auto-enable, but that was a bug and we no longer do it. But we're actually fixing the lack of auto-enabling.

Also, “no other unauthenticated cipher” should be “no unauthenticated cipher”, since CMAC is not an unauthenticated cipher.

Suggested wording:

> Fix the build when MBEDTLS_PSA_CRYPTO_CONFIG is enabled and the built-in CMAC is enabled, but no built-in unauthenticated cipher is enabled.